src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -907,7 +974,9 @@ void ImageReplayer<I>::handle_process_entry_safe(const ReplayEntry& replay_entry
     return;
   }
 
-  m_remote_journaler->committed(replay_entry);
+  if (m_remote_journaler) {
```

Comment:
Just wondering if you hit a case where the replayer gave you a callback after ```m_remote_journaler``` was destroyed?